### ISSUE TITLE
unused argument (reshape = TRUE)

### ISSUE THREAD

**Issue by asheetal** [mlr-org/mlr3#837]
### Description

Unsure how to fix this error on predict
```
Error in self$model$predict(data = as.matrix(newdata), reshape = TRUE) : 
  unused argument (reshape = TRUE)
```

### Reproducible example

library(mlr3)
library(mlr3learners.lightgbm)
library(tidyext)

Sys.setenv(CUDA_VISIBLE_DEVICES=1)

titanic_imputed$survived <- as.factor(titanic_imputed$survived)
num_classes <- length(unique(titanic_imputed$survived))
df <- onehot(titanic_imputed, 
             var = c("gender", "class", "embarked"),
             keep.original = FALSE)
task_classif <- as_task_classif(x = df, target = "survived")
learner_classif = lrn("classif.lightgbm", 
              objective = "multiclass", 
              device_type = "gpu",
              predict_type = "prob",
              early_stopping_round = 10L,
              learning_rate = 0.1,
              seed = 17L,
              metric = "multi_logloss",
              num_iterations = 100,
              num_class = num_classes,
              max_bin = 63L,
              num_leaves = 255L,
              tree_learner = "serial",
              min_data_in_leaf = 1L,
              min_sum_hessian_in_leaf = 100,
              num_threads=32L)

learner_classif$train(task = task_classif)
learner_classif$predict_newdata(task = task_classif, newdata = df)

**Comment by asheetal** [comment]
Did not realize it was an old package. Migrated to mlr3extralearners and resolved it
